etc/deploy/aws/aws-doco-script.sh

```diff
@@ -0,0 +1,279 @@
+#!/bin/bash
+
+# Preq: Run aws config to configure auth context profile before running this script.
+
+set -xeou pipefail
+
+jq --version || echo "This script required jq, install jq from https://stedolan.github.io/jq/download/"
+psql --version || echo "This script required psql, install psql from https://www.postgresql.org/download/"
+
+GLOBAL_TAG="jose-testing-4"
+
+CLUSTER_NAME="jose-pachyderm-cluster-4"
+AWS_REGION="us-east-2"
+AWS_PROFILE="default"
+
+BUCKET_NAME="jose-temp-bucket-4"
+S3_BUCKET_IAM_POLICY_NAME="jose-temp-bucket-iam-policy-4"
+S3_BUCKET_IAM_ROLE_NAME="jose-temp-bucket-iam-role-4"
+
+PODS_BUCKET_ACCESS_IAM_SA="jose-pods-bucket-access-iam-sa-4"
+
+EBS_CSI_DRIVER_POLICY_NAME="JoseAmazonEKS_EBS_CSI_Driver_Policy-4"
+
+POSTGRES_SQL_ID="jose-pachyderm-postgresql-4"
+DB_SUBNET_GROUP_NAME="jose-pachyderm-db-public-subnet-group-4"
+
+POSTGRES_SQL_DB_NAME_1="pachyderm"
+POSTGRES_SQL_DB_NAME_2="dex"
+POSTGRES_SQL_DB_USER_NAME="josepachyderm"
+POSTGRES_SQL_DB_USER_PASSWORD="josepachyderm"
+
+POSTGRES_SQL_DB_APP_USER_NAME="josepachydermapp"
+
+# Create cluster
+eksctl create cluster --name ${CLUSTER_NAME} --region ${AWS_REGION} --profile ${AWS_PROFILE} --tags Key=Name,Value=${GLOBAL_TAG}
+
+# Verify deployment
+kubectl get all
+
+# Create an S3 object store bucket for data
+LOCATION=$(aws s3api create-bucket --bucket ${BUCKET_NAME} --region ${AWS_REGION} --create-bucket-configuration LocationConstraint=${AWS_REGION})
```

Comment:
I do not see `$LOCATION` referred to elsewhere in the script.

Reply:
yea its just so aws's cli annoying pager doesn't hang on user input and allow it to continue. otherwise block the script from continuing.